### ISSUE TITLE
documentation on how to connect to the DB with a GUI.

### ISSUE THREAD

**Issue by alphex** [kalabox/kalabox#968]
I'm just stuck on the default vagrant@_hostname_ password question when trying to connect SEQUEL (for OSX) to the kalabox DB.

OR, if you have this documented somewhere, could you share?

Thanks!

**Comment by pirog** [comment]
Sounds likes maybe you are using the old Kalabox?

On Thursday, February 4, 2016, Duran Goodyear <notifications@github.com>
wrote:

> I'm just stuck on the default vagrant@*hostname* password question when
> trying to connect SEQUEL (for OSX) to the kalabox DB.
>
> OR, if you have this documented somewhere, could you share?
>
> Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kalabox/kalabox/issues/968>.
>


-- 
Mike Pirog
Co-Founder, Kalabox Inc

**Comment by alphex** [comment]
I'm on .11.1, latest release (just downloaded and installed yesterday).

Following directions I found here : https://kalamuna.atlassian.net/wiki/display/KALA/connecting+to+remote+MySQL+servers ... replacing SSH HOST with my value... It accepts everything, but asks for the password for "vagrant@_hostname_"

**Comment by alphex** [comment]
screen shot : http://z.alphex.io/2016-02-04_23-17-08.jpg

**Comment by pirog** [comment]
Those docs are pretty old and def for the old Kalabox. I would look in the
github wiki and issues.

**Comment by alphex** [comment]
I figured, and after searching for "SSH" and "Password", I opened this ticket.

**Comment by pirog** [comment]
There is a ticket that has the workaround to connect to MySQL. On phone so
hard to direct you there now.

Also should be tickets to create a MySQL client wrapper and a command to
easily expose connection info

**Comment by labboy0276** [comment]
Hello!

Here is the workaround I found:

https://github.com/kalabox/kalabox/issues/899

**Comment by alphex** [comment]
@labboy0276 if I can bug you.

Followed your directions in #899 and "kbox containers > tmp.txt" doesn't put anything in to the file, and "kbox containers | grep "HostPort" doesn't print anything on screen.

http://z.alphex.io/2016-02-05_14-03-04.jpg

I edited the two files as shown.
Saved,  then did a "kbox stop && kbox rebuild && kbox start"

The domain "db.klip-master.kbox" resolves.   

Thoughts?

**Comment by labboy0276** [comment]
Hello

Are you in the app directory itself when you run those commands?

**Comment by alphex** [comment]
@labboy0276 I am.

> alphex:klip-master duran$ pwd
/Users/duran/dev/kalabox/klip-master
alphex:klip-master duran$ ls -la
total 48
drwxr-xr-x  13 duran  staff   442 Feb  5 14:14 .
drwxr-xr-x   5 duran  staff   170 Feb  5 13:29 ..
-rw-r--r--   1 duran  staff  3279 Feb  5 13:29 cli.yml
drwxr-xr-x  33 duran  staff  1122 Feb  5 13:31 code
drwxr-xr-x  11 duran  staff   374 Feb  5 13:29 config
drwxr-xr-x   8 duran  staff   272 Feb  5 13:29 dockerfiles
-rw-r--r--   1 duran  staff  3464 Feb  5 13:29 kalabox-cli.yml
-rw-r--r--   1 duran  staff  6879 Feb  5 13:58 kalabox-compose.yml
-rw-r--r--   1 duran  staff   728 Feb  5 13:58 kalabox.yml
drwxr-xr-x  40 duran  staff  1360 Feb  5 13:29 node_modules
-rw-r--r--   1 duran  staff   176 Feb  5 13:29 package.json
drwxr-xr-x   3 duran  staff   102 Feb  5 13:29 plugins
-rw-r--r--   1 duran  staff     0 Feb  5 14:00 tmp.txt
alphex:klip-master duran$ kbox containers | grep "HostPort"
alphex:klip-master duran$

**Comment by pirog** [comment]
Make sure you export port 3306 in your compose file and rebuild.

**Comment by labboy0276** [comment]
@pirog he did.  @alphex what happens when you type kbox containers

**Comment by alphex** [comment]
@labboy0276 nothing, just returns me to the prompt,  no results.

sending it to your "afilename.txt" doesn't write anything to the file either.

**Comment by labboy0276** [comment]
something is off then, it should spit out a giant array.

Do a kbox version and see what it prints out

**Comment by alphex** [comment]
@labboy0276 

$ kbox version
0.11.1

**Comment by labboy0276** [comment]
something isn't right then.

Run this:
```
cd ~/.kalabox/bin
./docker-machine ssh Kalabox2
docker ps -a
```
What do you see?

**Comment by alphex** [comment]
@labboy0276 

http://z.alphex.io/2016-02-05_16-07-55.jpg

I'm only trying out your workaround in the klip-master environment.
(You'll see I've been trying it out with a few other environments)

**Comment by labboy0276** [comment]
Well klip collective isnt in there.  so that is the issue, it doesnt exist in the VB.

Lets try this then:

cd into the collective app:
kbox destroy, kbox rebuild, kbox pull, kbox restart

then try my work around.

**Comment by alphex** [comment]
@labboy0276 umm, weird.

right before I ran your command, I did this.

> alphex:klip-master duran$ kbox stop && kbox rebuild && kbox start

Just to make sure it was running.

I will destroy, rebuild, pull, restart now.

**Comment by alphex** [comment]
@labboy0276 
Completely uninstalled kalabox (ran the uninstall scripts).
Re-installed.

re-created my "klip-master" environment.
Followed you're directions again.

Edited kalabox.yml and kalabox-compose.yml as you defined.

Saved files,  then "kbox stop && kbox rebuild && kbox start".

During the start portion, I got a syncthing start error, indicating a time out.

> Unhandled rejection VError: Error dispatching event listeners [pre-app-start]: Unexpected error while waiting for syncthing http://127.0.0.1:60008 to start.: operation timed out
    at /usr/local/bin/lib/core/events.js.jx:146:11

I ran "kbox start" again on its own, and it came up, and I could load the site in my browser, so I know everything is working.

... So I go back to run "kbox containers | grep "HostPort""  and it doesn't return anything.

"kbox containers" returns nothing either.

Thanks for your help.
Thoughts?

**Comment by labboy0276** [comment]
Not sure then, @pirog any thoughts on this one?

**Comment by labboy0276** [comment]
Quick question though.  Before we were trying to see what was wrong with klip collective, but now you are mentioning klip master.  So I maybe somewhat confused.  Which site are you trying to win here?

Also, we could try a complete uninstall, try downloading the bin and uninstalling all the things, rm -rf ~/kalabox & ~/.docker , restart your machine and try again.

**Comment by labboy0276** [comment]
@alphex so I just noticed this.  When I am in my app and I type kbox containers now, it returns nothing.  However, if I am not in my app, just say in my dowblaods folder, when I type kbox containers I see the data....

Sorry about this, as I just noticed this.  Hopefully you figured out your issue.

@pirog is it a issue where you are in an app and you enter kbox containers and it returns nothing?  Or is that working as designed?
